src/hooks/usePokemons.ts

```diff
@@ -35,15 +36,7 @@ const usePokemons = () => {
 
   const pokemonIndex =
     data?.pokemon_v2_pokemon.map((item) => {
```

Comment:
super nit: could be just:
```suggestion
    data?.pokemon_v2_pokemon.map(parsePokemon)
```